site/posts/jupyter-notebooks-faq.ipynb

```diff
@@ -0,0 +1,195 @@
+    "\n",
+    "    ```\n",
+    "2. Try uninstalling and re-installing Conda, making sure to answer “YES” this time,\n",
+    "\n",
+    "-----\n",
+    "<!-- TEASER_END -->"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** I executed `jupyter lab` and received the message, “Could not determine jupyterlab build status without nodejs”. Do I need to exit somehow and or fix something?\n",
+    "\n",
+    "**A.** This is only a warning, not an error message. No fix is necessary. \n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** How do you specify what browser Jupyter opens with?\n",
+    "\n",
+    "**A.** If you don’t want to use your default browser (which Jupyter will automatically open with), you can specify a different one with `jupyter lab —browser=chrome` \n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Jupyter opens to my base environment, not the environment I launched `jupyter lab` from. Is this an error?\n",
+    "\n",
+    "**A.** This is the expected behavior. You can double check your environment by typing `conda env list` from your Jupyter terminal to see all of your environments. The one with the asterisks next to it is your activated environment.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** The Jupyter workspace autocompletion of quotation marks and parenthesis is causing errors. Is there a way to disable this?\n",
+    "\n",
+    "**A.** Yes according to this this [stack overflow post](https://stackoverflow.com/questions/44216326/how-to-disable-auto-quotes-and-auto-brackets-in-jupyter-5-0#:~:text=2%20Answers&text=After%20executing%20the%20Python%20command,auto%2Dclosing%20quotes%20and%20brackets), in any code cell type: \n",
+    "\n",
+    "```\n",
+    "from notebook.services.config import ConfigManager\n",
+    "c = ConfigManager()\n",
+    "c.update('notebook', {\"CodeCell\": {\"cm_config\": {\"autoCloseBrackets\": False}}})\n",
+    "```\n",
+    "\n",
+    "You may have to restart the Jupyter session for the changes to take effect.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Is converting a cell from code to raw a good way to basically “comment out” an entire cell?  Is that good practice or bad practice?\n",
+    "\n",
+    "**A.** This is the easiest way to \"comment out\" a cell.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Equations aren’t working in my markdown cells. What should I do?\n",
+    "\n",
+    "**A.** Double check that you've used `$$` to bookend your equation.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
```

Comment:
I would delete this question.  It does not have enough specificity to be useful, and the answer reflects that.

```suggestion
```